File: Electronics_Shop_17.Model/Requests/UpdateSeller.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Model.Requests
{
    public class UpdateSeller
    {
        public string? StoreName { get; set; }
        public int? LicenseNumber { get; set; }
        public virtual UpdateAdress? Adress { get; set; }
        public int? UserAccountId { get; set; }
        public int? PersonId { get; set; }
        public bool? isDeleted { get; set; }

    }
}

File: angular17_web/src/app/models/seller/update.seller.ts
import { UpdateAdress } from '../adress/update.adress';

export class UpdateSeller {
  storeName?: string;
  licenseNumber?: number;
  adress?: UpdateAdress;
  userAccountId?: number;
  personId?: number;
  isDeleted?: boolean;

  constructor(data?: any) {
    if (!data) return;

    this.storeName = data.storeName;
    this.licenseNumber = data.licenseNumber;
    this.adress = data.adress ? new UpdateAdress(data.adress) : undefined;
    this.userAccountId = data.userAccountId;
    this.personId = data.personId;
    this.isDeleted = data.isDeleted;
  }
}
